packages/edit-navigation/src/components/editor/style.scss

```diff
@@ -1,43 +1,103 @@
-	flex-grow: 1;
+	background: $white;
+	border: $border-width solid $gray-900;
+	border-radius: $radius-block-ui;
+	margin-top: $grid-unit-50 * 2;
 
-	@include break-medium() {
-		align-items: flex-start;
-		display: flex;
+	.components-spinner {
+		display: block;
+		margin: $grid-unit-15 auto;
 	}
-}
 
-.edit-navigation-editor__block-view {
-	padding: $grid-unit-20;
-	@include break-medium() {
-		flex-grow: 1;
-		align-self: stretch;
+	// The following attempts to adapt the layout of the Navigation
+	// and Link blocks to work better in the context of the Navigation Screen.
+	.wp-block-navigation {
+		margin: 0;
+		font-size: 15px;
+		padding: $grid-unit-15;
 	}
 
-	.components-spinner {
+	.wp-block-navigation-link {
 		display: block;
-		margin: 10px auto;
+
+		// The following attempts to fix the focus outlines
+		&.is-selected > .wp-block-navigation-link__content,
+		&.is-selected:hover > .wp-block-navigation-link__content {
+			box-shadow: 0 0 0 var(--wp-admin-border-width-focus) var(--wp-admin-theme-color);
+		}
+
+		&.block-editor-block-list__block:not([contenteditable]):focus::after {
+			display: none;
+		}
 	}
-}
 
-.edit-navigation-editor__list-view {
-	border-top: 1px solid $gray-300;
-	padding: $grid-unit-20;
+	.wp-block-navigation-link__content {
+		padding: 0;
+		margin-bottom: 6px;//$grid-unit-05;
+		border-radius: $radius-block-ui;
 
-	@include break-medium() {
-		border-left: 1px solid $gray-300;
-		border-top: none;
-		margin-top: 0;
-		width: 300px;
+		&:hover {
+			box-shadow: 0 0 0 $border-width $gray-300;
+		}
 	}
 
-	.edit-navigation-editor__list-view-title {
-		font-size: 1.2em;
-		margin: 0;
+	.wp-block-navigation-link__label {
+		padding: $grid-unit-15;
+		padding-left: $grid-unit-30;
+
+		// Without this Links with submenus display a pointer.
+		cursor: text;
 	}
 
-	.components-spinner {
-		display: block;
-		margin: 100px auto;
+	// Position the submenu icon so it appears to the left of
+	// the Link. All the extra specificity is help override the
+	// rotation on the SVG.
+	.wp-block-navigation .wp-block-navigation-link .wp-block-navigation-link__submenu-icon {
+		position: absolute;
+		top: 6px;
+		left: 2px;
+		padding: 6px;
+		pointer-events: none;
+
+		svg {
+			padding: 0;
+			transform: rotate(90deg);
+		}
+	}
+
+	// Submenus
+	// There's a bunch of stuff to override just to get submenus to
+	// as a normal block element.
+	.wp-block-navigation-link.has-child {
+		cursor: default;
+		border-radius: $radius-block-ui;
+	}
+
+	// When editing a link with children, highlight the parent
+	// and adjust the spacing and submenu icon.
+	.wp-block-navigation-link.has-child.is-editing {
+		> .wp-block-navigation__container {
+			opacity: 1;
+			visibility: visible;
+			position: relative;
+			background: transparent;
+			top: auto;
+			left: auto;
+			padding-left: $grid-unit-15;
+			min-width: auto;
+			width: 100%;
+			border: none;
+			display: block;
+
+			&::before {
+				display: none;
+			}
+		}
+	}
```

Comment:
Actually, I remember now about the discussion around using a dropdown block for submenus, so maybe in some distant future the nav screen would use a collapse/expand block (I completely forget the proper name for this?) instead of dropdown.